Filters/exceptions.txt

```diff
@@ -2,6 +2,8 @@
 ! Once you enable private DNS, Android 9 starts resolving random domains looking like 
 ! `*-dnsotls-ds.metric.gstatic.com` (for instance, `a5a6380f-dnsotls-ds.metric.gstatic.com`).
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/32303
+@@||e.reddit.com^|

```

Comment:
better add original rule to exclusions.txt

Reply:
OK, done.
https://github.com/AdguardTeam/AdGuardSDNSFilter/pull/61